source/tutorial/install-mongodb-on-windows.txt

```diff
@@ -93,10 +98,11 @@ Manually Create a Windows Service for MongoDB Community Edition
 You can set up the MongoDB server as a :guilabel:`Windows Service` that
 starts automatically at boot time.
 
-The following procedure assumes you have installed MongoDB Community using the
-``.msi`` installer with the path ``C:\Program Files\MongoDB\Server\3.6\``.
+The following procedure assumes you have installed MongoDB Community 
+using the ``.msi`` installer with the path 
+``C:\Program Files\MongoDB\Server\{+version+}\``.
```

Comment:
The trailing backslash is inconsistent with other command line paths in the procedure, such as 'mkdir c:\data\db' in step 2.